### PR TITLE
Add assertion results based on variant. Removed authentication check.

### PR DESCRIPTION
**Pull request by theferrit32**

### CODE REVIEW

File: shadow-cljs.edn [hunk 1]
```diff
@@ -2,7 +2,11 @@
 {:source-paths ["src"]
  :dependencies [[re-frame "1.1.1"]
                 [re-graph "0.1.14"]
-                [metosin/reitit "0.5.10"]]
+                [metosin/reitit "0.5.10"]
+                [cljsjs/firebase "7.5.0-0"]
```
Comment (tnavatar) [inline shadow-cljs.edn:6]: I think that the cljsjs* packages are generally not used with shadow-cljs, is there a specific reason for this one?

File: package.json [hunk 2]
```diff
@@ -12,7 +13,8 @@
     "react-dom": "^16.12.0",
     "react-flip-move": "^3.0.1",
     "react-highlight.js": "1.0.7",
-    "shadow-cljs": "^2.11.7"
+    "shadow-cljs": "^2.11.18",
+    "xregexp": "^5.0.1"
```
Comment (tnavatar) [inline package.json:17]: I'm not seeing includes for this dependency--what is it used for?

File: src/curator/pages/home/views.cljs [hunk 3]
```diff
@@ -9,11 +9,14 @@
 
 (defn search []
   (let [topic @(subscribe [::subs/search-topic])
-        suggested-genes @(subscribe [::subs/suggested-genes])]
-    [:section.section.home-search-main 
+<<<<<<< HEAD
```
Comment (tnavatar) [inline src/curator/pages/home/views.cljs:12]: Looks like an artifact from a merge conflict slipped through...

File: src/curator/pages/home/views.cljs [hunk 4]
```diff
@@ -24,48 +27,114 @@
...
                           :placeholder topic
                           :on-change #(dispatch [:home/request-suggestions
-                                                 (-> % .-target .-value)])}]]]
-         (when (seq suggested-genes)
-           [:div.box
-            (for [gene suggested-genes]
-              [:div.block
-               [:a {:href (rfe/href :genes gene)}
-                (:text gene)]])])]]]]]))
+                                                 ;(fn [event] (println event) (-> event .-target .-value))
```
Comment (tnavatar) [inline src/curator/pages/home/views.cljs:50]: I think we'll probably want to split the variation display onto a separate page soon--but this is a starting point.

**Review by tnavatar — CHANGES_REQUESTED** [review]
I think the cljsjs package should be removed & see other comments.

File: package.json (hunk 2 above)
Comment (theferrit32) [reply package.json:17]: `cuerdas` requires the npm package `xregexp` and for some reason it isn't being pulled in automatically by shadow-cljs

**Pull request merged**